tests/receivers/jmxreceiver/jmxreceiver_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright Splunk, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build integration
+
+package tests
+
+import (
+	"os"
+	"path"
+	"testing"
+
+	"github.com/signalfx/splunk-otel-collector/tests/testutils"
+)
+
+func TestJMXReceiverProvidesAllJVMMetrics(t *testing.T) {
+	// JMX metrics are missing when running on arm so the test only checks a subset
+	// of what would be received otherwise.
```

Comment:
Is this documented anywhere that we should include in our docs?

Reply:
It's not documented anywhere that I've found. The only jmx collector [documentation from Splunk](https://docs.splunk.com/Observability/gdi/monitors-languages/jmx.html) that I've seen is pretty out of date, so there isn't a good spot to update there either. It's also not documented in the contrib repo, nor the java jmx metrics gatherer repo.

Reply:
In other instances where this is the case I have a feeling it's related to qemu and not an actual issue w/ arm. Once we have real arm instances in CI my assumption can be validated: https://github.com/signalfx/splunk-otel-collector/blob/f6dc932544b394f09635685afb57d66ec0069d1a/tests/receivers/discovery/host_observer_test.go#L28